Willning/DLOplanTool
# Dlo 57

### **Related Issue/Keyword:**
[DLO-57](https://trello.com/c/tgOLveQd/57-notifications-popup-problem-fix)
### **Description:**
use @material-ui/core/Table component to handle page changing rather than using react-bootstrap Pagination to writing custom handlePageChange function
### **Testing:**
**Steps for manual testing:**
1. need to have more than 6(more is better) projects shared with you
2. open the "bell" icon on the top left, you will see a list of sharing notices, the pagination items on the bottom of the table should work properly,  the default "rows per page" is 10, 
![image](https://user-images.githubusercontent.com/3829984/65760968-1537a380-e172-11e9-84e5-b12da76dacb5.png)


### **Checklist:**
